### ISSUE TITLE
Question about training few- and zero-shot setting

### ISSUE THREAD

**Issue by packer-c** [YuanYuan98/UniFlow#1]
Super nice work!

Could you tell me how to set up and train with few- and zero-shot in your code?
Thanks in advance!

**Comment by YuanYuan98** [comment]
To set up and train with few- and zero-shot scenarios, you can adjust the hyperparameter  `few_ratio`. This parameter controls the proportion of training data used for model training. You can find its implementation in the  [Dataloader](https://github.com/YuanYuan98/UniFlow/blob/main/src/DataLoader.py#L162-L163).

**Comment by packer-c** [comment]
Got it, thanks for your kindly reply!

Another question is only for few-shot scenario, where can I find the finetune process in your code?

**Comment by YuanYuan98** [comment]
UniFlow employs an end-to-end training approach where the loss is directly computed as the prediction loss. Unlike [UniST](https://github.com/tsinghua-fib-lab/UniST), there is no separate pre-training or fine-tuning process. For few-shot or zero-shot scenarios, you simply need to adjust the proportion of the specific training dataset to control the data used.

**Comment by packer-c** [comment]
Clear, great!
